### DISCUSSION TITLE
[Feature] Maps Setting: "Include Shared Albums" on Mobile

### DISCUSSION THREAD

**Discussion by clembuck** [immich-app/immich#12571] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

The "Include Shared Albums" option is available in the web version:
![map_pc](https://github.com/user-attachments/assets/cf59e47d-f59a-4e5c-a367-c7976e9a5e88)

However, it appears to be missing on the mobile version:
![map_android](https://github.com/user-attachments/assets/3ba5d0b3-c96b-4742-8cdb-16949e6e60c3)

I would love to see this feature added to the mobile app

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by haoenz** [comment]
In addition, the mobile version of the map view does not seem to be able to customize the time range.
I am a little curious why this module is different between the web version and the mobile version.
